<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AreaRobotWidget</class>
 <widget class="QWidget" name="AreaRobotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,1">
   <item>
    <widget class="QLabel" name="label_42">
     <property name="text">
      <string>Distance</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_20">
     <item>
      <widget class="QSlider" name="distanceSlider">
       <property name="maximum">
        <number>333</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="distanceRealLabel">
       <property name="text">
        <string>(0,00 m)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="distanceBox">
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_41">
     <property name="text">
      <string>Angle</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <widget class="QSlider" name="angleSlider">
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="angleBox">
       <property name="suffix">
        <string>Â°</string>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_33">
     <property name="text">
      <string>Trigger distance</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QSlider" name="triggerSlider">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="triggerRealLabel">
       <property name="text">
        <string>(0,00 m)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="triggerBox">
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ShockRobotLabel" name="shockLocationLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShockRobotLabel</class>
   <extends>QLabel</extends>
   <header>gui/shockrobotlabel.h</header>
   <slots>
    <slot>setAngle(int)</slot>
    <slot>setDistance(int)</slot>
    <slot>setRadius(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>triggerSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>triggerBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>triggerBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>triggerSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angleBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>angleSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>angleBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>distanceBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>distanceSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>distanceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>distanceBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angleBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>shockLocationLabel</receiver>
   <slot>setAngle(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>distanceBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>shockLocationLabel</receiver>
   <slot>setDistance(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>triggerBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>shockLocationLabel</receiver>
   <slot>setRadius(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
